README.md

```diff
@@ -40,93 +40,72 @@
 
 ## 🎮 Demo
 
-Give a try to the [__⏩ demo live environment__][demo-live-environment].
-The demo let you load a BPMN file to see how `bpmn-visualization` renders it. Various versions of the lib are available. 
+Please check [__⏩ demo live environment__](https://cdn.statically.io/gh/process-analytics/bpmn-visualization-examples/master/examples/index.html). \
```

Comment:
:question: switch title to `examples` for consistency with the description and other documentation elements?

Reply:
when I check some new lib I always ctrl+f 'demo' to quick search the preview
but it is subjective
maybe 'Demo examples'